Makefile.am

```diff
@@ -339,6 +337,9 @@ libsqlite3_la_SOURCES = sqlite3.c
 libsqlite3_la_CFLAGS = -g3
 
 unit_test_LDADD += libsqlite3.la
+if BUILD_RAFT_ENABLED
+raft_core_unit_test_LDADD += libsqlite3.la
```

Comment:
I think tests that use libsqlite3 should be in the dqlite part of the tree.

Reply:
We are using sqlite to create the hash-table, it does not have anything to do with dqlite per se. The raft code itself (code for creating the ht) has a dependency on SQLite, not only the tests. We could abstract it away to maintain Raft more agnostic but I am not sure there is a point any more in the extra complexity.